Sources/ExyteChat/ChatView/MessageView/MessageMenu/MessageMenu.swift

```diff
@@ -267,7 +267,6 @@ struct MessageMenu<MainButton: View, ActionEnum: MessageMenuAction>: View {
             )
             
             messageTopPadding = cellFrame.height - messageFrame.height
```

Comment:
The new VStack + VStack method causes this calculation to no longer make sense. If we change this to a static 4px the message menu seems to line up correctly with the underlying message and results in a smoother animation / transition.
```swift
- messageTopPadding = cellFrame.height - messageFrame.height
+ messageTopPadding = 4
```

Reply:
Yes. It helps. It still jiggles a little when the modal screen appears, but it looks smooth in slow animation mode. But it doesn't seem like it connected with our changes.